test/topology/test_null.py

```diff
@@ -5,23 +5,18 @@
 #
 from cassandra.protocol import InvalidRequest   # type: ignore
 from pylib.util import unique_name
+from pylib.schema_helper import get_schema
 import pytest
 
 
-@pytest.fixture(scope="session")
-async def table1(cql, keyspace):
-    table = keyspace + "." + unique_name()
-    cql.execute(f"CREATE TABLE {table} (p text, c text, v text, primary key (p, c))")
-    yield table
-    await cql.run_async("DROP TABLE " + table)
-
-
 @pytest.mark.asyncio
-async def test_delete_empty_string_key(cql, table1):
+async def test_delete_empty_string_key(cql):
+    tables = await get_schema("delete_empty_string_key", cql, ntables=1, ncolumns=5)
     s = "foobar"
     # An empty-string clustering *is* allowed:
-    await cql.run_async(f"DELETE FROM {table1} WHERE p='{s}' AND c=''")
+    await cql.run_async(f"DELETE FROM {tables[0]} WHERE pk = '{s}' AND {tables[0].columns[1].name} = ''")
     # But an empty-string partition key is *not* allowed, with a specific
     # error that a "Key may not be empty":
     with pytest.raises(InvalidRequest, match='Key may not be empty'):
-        await cql.run_async(f"DELETE FROM {table1} WHERE p='' AND c='{s}'")
+        await cql.run_async(f"DELETE FROM {tables[0]} WHERE pk = '' AND {tables[0].columns[1].name} = '{s}'")
+    await tables.verify_schema()
```

Comment:
Again, I don't understand what this tables.verify_schema() is supposed to be doing or testing in a test called test_delete_empty_string_key.
Also where does the test table get deleted here?

Reply:
I'm still waiting to understand why test_null.py and its test_delete_empty_string_key even exists. Why not remove it and start with a real test?

Reply:
test_null was removed in v8